images/public-log-vendor-matcher/pg-init.d/lib-06-bq-load-logs.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Load the k8s-infra-artifacts-gcslogs logs into the usage_all_raw dataset
+if [ -z "${GCP_BIGQUERY_DATASET_LOGS:-}" ]; then
+    echo "Using dataset logs, since \$GCP_BIGQUERY_DATASET_LOGS was provided and set to '$GCP_BIGQUERY_DATASET_LOGS'"
+    BUCKETS=(
```

Comment:
We should be including all of the staging buckets as well

Reply:
Where is a list of the buckets?

Reply:
`ls -1 audit/projects/*/buckets/ | grep ^k8s-staging`

Reply:
that misses the gcr buckets but close enough to start